components/multiselect/multiselect.jsx

```diff
@@ -258,6 +263,7 @@ export default class MultiSelect extends React.Component {
                             disabled={this.props.saving}
                             onClick={this.handleOnClick}
                             defaultMessage={buttonSubmitText}
+                            savingMessage={buttonSubmitLoadingText}
```

Comment:
This can actually just be `savingMessage={this.props.buttonSubmitLoadingText}` without the above changes. We only have the other stuff for `buttonSubmitText` because it's sometimes set differently